instrumentation/gwt-2.0/javaagent/gwt-2.0-javaagent.gradle

```diff
@@ -0,0 +1,93 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "com.google.gwt"
+    module = "gwt-servlet"
+    versions = "[2.0.0,)"
+    assertInverse = true
+  }
+}
+
+sourceSets {
+  testapp {
+    java
+    resources {
+      srcDirs("src/webapp")
+    }
+    java.outputDir = file("$buildDir/testapp/classes")
+    compileClasspath += sourceSets.main.compileClasspath
+  }
+}
+
+dependencies {
+  // these are needed for compileGwt task
+  if (findProperty('testLatestDeps')) {
+    compileOnly 'com.google.gwt:gwt-user:+'
+    compileOnly 'com.google.gwt:gwt-dev:+'
+  } else {
+    compileOnly 'com.google.gwt:gwt-user:2.0.0'
+    compileOnly 'com.google.gwt:gwt-dev:2.0.0'
+  }
+
+  library 'com.google.gwt:gwt-servlet:2.0.0'
+
+  testInstrumentation project(':instrumentation:servlet:servlet-3.0:javaagent')
+  testInstrumentation project(':instrumentation:servlet:servlet-javax-common:javaagent')
+  testInstrumentation project(':instrumentation:jetty-8.0:javaagent')
+
+  testImplementation "org.testcontainers:selenium:${versions.testcontainers}"
+  testImplementation 'org.seleniumhq.selenium:selenium-java:3.141.59'
+
+  testImplementation(project(':testing-common')) {
+    exclude group: 'org.eclipse.jetty', module: 'jetty-server'
+  }
+  testImplementation group: 'org.eclipse.jetty', name: 'jetty-webapp', version: '9.4.35.v20201120'
+}
+
+task copyTestWebapp(type: Copy) {
```

Comment:
Wouldn't it be easier to have a separate module for test app?

Reply:
Assuming that we want to compile the test app with the same version of gwt that we use to run the test even if it is doable having the test app in another module would probably make this even more confusing.

Reply:
As for the server span name suggestions are welcome. Currently it is the servlet mapping where the service is mapped as far as I can tell each service should be mapped to a different path so it should be a rather good name. Alternative would be to set server span to invoked method name, but then server span and nested span would have the same name.